docs/en/automation/crmscript/reference/CRMScript.Global.Void.yml

```
@@ -567,6 +568,32 @@ items:
     return:
       type: CRMScript.Global.Void
   example: 
+- uid: CRMScript.Global.Void.authenticateCustomer(String,String)
+  commentId: M:CRMScript.Global.Void.authenticateCustomer(String,String)
+  id: 'authenticateCustomer(String,String)'
+  so.intellisense: Void.authenticateCustomer
+  so.version: 10.2.25
```

Did you mean .5 and not .25?

```suggestion
  so.version: 10.2.5
```